### ISSUE TITLE
Mustache API call to extract key names?

### ISSUE THREAD

**Issue by bluemont** [mustache/mustache#134]
I asked [this question on Stack Overflow](http://stackoverflow.com/questions/10920826/extract-key-names-using-mustache-ruby-api) first, but now I think I should ask it over here:

```
m = Mustache.new
m.template = "Hello {{first_name}} {{last_name}}"
```

I want to make an API call -- but I don't know what it is -- that returns the key names:

```
[:first_name, :last_name]
```

From what I can tell, this is not supported.

**Comment by meganemura** [comment]
@locks 

This feature is already merged at #143.

```ruby
Mustache::Template.new(template).tags
```

I think we can close this issue.

**Comment by locks** [comment]
@meganemura thank you for the heads up!
